core/test/user-profile.test.ts

```diff
@@ -133,49 +132,6 @@ describe('UserProfile cookie tests', () => {
       await tab.close();
     }
   });
-
-  it('restores cookies for safari', async () => {
```

Comment:
Do you think we'll just throw out this test? Or would it move into the safari repo?

Reply:
I moved it into the safari repo.